### DISCUSSION TITLE
Need help with 1xBetCrashupdater

### DISCUSSION THREAD

**Discussion by codedblack47** [aa-sikkkk/FortuneTeller#3] [Q&A]
Sorry for the stress but I'm new with this selenium and css stuff.

I'm trying all that I can to find the exact css selector to extract the multiplier from the defined 1xbet url.

Any help will be greatly appreciated 

Thanks in advance

**Comment by aa-sikkkk** [comment]
> Sorry for the stress but I'm new with this selenium and css stuff.
> 
> I'm trying all that I can to find the exact css selector to extract the multiplier from the defined 1xbet url.
> 
> Any help will be greatly appreciated 
> 
> Thanks in advance

I too couldn't find the exact selector!! If you happen to find it, update the script and create a pull request❤️

**Reply by codedblack47** [reply]
Noted

**Reply by JEAZY-LEKE** [reply]
have you found a solution to this

**Comment by codedblack47** [comment]
Nope., gave up

**Reply by JEAZY-LEKE** [reply]
same bro
